.github/workflows/nodejs.yml

```diff
@@ -4,11 +4,9 @@ on:
   push:
     branches:
       - master
-      - next
   pull_request:
     branches:
       - master
-      - next
```

Comment:
Don't touch it, we will use `next` in future

Reply:
reverted